pkg/client/docs/models/components/v3currencycloudconfig.md

```
@@ -3,12 +3,12 @@
-| `Name`             | *string*           | :heavy_check_mark: | N/A                |
-| `PageSize`         | **int64*           | :heavy_minus_sign: | N/A                |
-| `PollingPeriod`    | **string*          | :heavy_minus_sign: | N/A                |
-| `Provider`         | **string*          | :heavy_minus_sign: | N/A                |
\ No newline at end of file
+| Field                                                                   | Type                                                                    | Required                                                                | Description                                                             |
+| ----------------------------------------------------------------------- | ----------------------------------------------------------------------- | ----------------------------------------------------------------------- | ----------------------------------------------------------------------- |
+| `APIKey`                                                                | *string*                                                                | :heavy_check_mark:                                                      | N/A                                                                     |
+| `Endpoint`                                                              | *string*                                                                | :heavy_check_mark:                                                      | N/A                                                                     |
+| `LoginID`                                                               | *string*                                                                | :heavy_check_mark:                                                      | N/A                                                                     |
+| `Name`                                                                  | *string*                                                                | :heavy_check_mark:                                                      | N/A                                                                     |
+| ~~`PageSize`~~                                                          | **int64*                                                                | :heavy_minus_sign:                                                      | : warning: ** DEPRECATED **: From v3.1, this parameter will be ignored. |
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)